plugin/src/gatsby-node.ts

```diff
@@ -74,7 +74,9 @@ async function sourceAllNodes(
 
 const shopifyNodeTypes = [
   `ShopifyLineItem`,
-  `ShopifyMetafield`,
+  `ShopifyProductMetafield`,
+  `ShopifyProductVariantMetafield`,
+  `ShopifyCollectionMetafield`,
```

Comment:
Wondering if there's any way to get these node types through the Gatsby API, which would prevent this bug from happening again and eliminate some maintenance. If we have defined these types in the `schema`, then ideally the `schema` could also return information about the types that a plugin has defined.

Couldn't find anything like that though. @TylerBarnes do you know of a way to do that?